lib/features/connect/Connect.js

```diff
@@ -76,24 +107,23 @@ export default function Connect(eventBus, dragging, modeling, rules) {
    * Start connect operation.
    *
    * @param {DOMEvent} event
-   * @param {djs.model.Base} source
-   * @param {Point} [sourcePosition]
-   * @param {Boolean} [autoActivate=false]
+   * @param {djs.model.Base} start
+   * @param {Point} [connectionStart]
+   * @param {boolean} [autoActivate=false]
```

Comment:
Is there some pattern that it should be `{boolean}` rather than `{Boolean}`? If so, we should fix this in a single commit, across our whole code base, rather than fixing it one by one.

Reply:
Primitive types start with a lowercase letter. See https://2ality.com/2011/08/jsdoc-intro.html

Reply:
In this case we should consider to batch-migrate _all_ of our type definitions.